build.sbt

```diff
@@ -0,0 +1,34 @@
+import Dependencies._
+
+ThisBuild / scalaVersion     := "2.13.8"
+ThisBuild / version          := "0.1.0-SNAPSHOT"
+ThisBuild / organization     := "uk.gov.nationalarchives"
+ThisBuild / organizationName := "The National Archives"
+ThisBuild / startYear := Some(2023)
+ThisBuild / licenses := Seq("MIT" -> url("https://opensource.org/licenses/MIT"))
+ThisBuild / description := "JMS4S Request-Reply Library"
+
+/* IMPORTANT NOTE >>>
+ * Until further notice you will need to sbt publishLocal the JMS4S library from the combined-features branch
+ * of this repository https://github.com/rwalpole/jms4s
+ */
+
+lazy val root = (project in file("."))
+  .enablePlugins(AutomateHeaderPlugin)
+  .settings(
+    name := "jms4s-request-reply",
+    libraryDependencies ++= Seq(
+      "org.typelevel" %% "cats-core" % "2.9.0",
+      "org.typelevel" %% "cats-effect" % "3.4.4",
+      "org.typelevel" %% "cats-effect-kernel" % "3.4.4",
+      //"org.typelevel" %% "log4cats-slf4j" % "2.5.0",
```

Comment:
Logging doesn't work on my machine, I get:
```
SLF4J: No SLF4J providers were found.
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See https://www.slf4j.org/codes.html#noProviders for further details.
```
Is this something that we can sort out by adding a dependency?